src/components/CartList.js

```diff
@@ -1,72 +1,106 @@
-              <Item key={item.id}>
-                <ProductDetails>
-                  <ProductImage
-                    src={item.images[0]}
-                    alt={item.name}
-                  ></ProductImage>
-                  <div>
-                    <p>{item.name}</p>
-                    <p>Product code: {item.id}</p>
-                  </div>
-                </ProductDetails>
-                <td>
-                  <ProductQuantity>
-                    <input
-                      type="number"
-                      value={item.quantity}
-                      label="quantity"
-                    ></input>
-                    <RemoveBtn>X</RemoveBtn>
-                  </ProductQuantity>
-                </td>
-                <td>{item.price}</td>
-                <td>{item.price * item.quantity}</td>
-              </Item>
-            ))}
-          </tbody>
-        </table>
-      </CartListContainer>
-      <Summary>
-        <h3>Summary</h3>
-        <hr />
-        <p>Items: {items.length}</p>
-        <p>Total cost: ${total}</p>
-        <Button>Checkout</Button>
-      </Summary>
+        <CartListContainer>
+          {items.length ? (
+            <Table>
+              <thead>
+                <tr>
+                  <th>Product details</th>
+                  <th>Quantity</th>
+                  <th>Price</th>
+                  <th>Total</th>
+                </tr>
+              </thead>
+              <tbody>
+                {items.map((item) => (
+                  <Item key={item.id}>
+                    <ProductDetails>
+                      <ProductImage
+                        src={item.images[0]}
+                        alt={item.name}
+                      ></ProductImage>
+                      <div>
+                        <p>{item.name}</p>
+                        <p>Product code: {item.id}</p>
+                      </div>
+                    </ProductDetails>
+                    <td>
+                      <ProductQuantity>
+                        <input
+                          type="number"
+                          value={item.quantity}
+                          label="quantity"
+                          min="0"
+                          onChange={handleQuantityChange(item.id)}
+                        ></input>
+                        <RemoveBtn onClick={handleRemove(item.id)}>X</RemoveBtn>
+                      </ProductQuantity>
+                    </td>
+                    <td>{item.price}</td>
+                    <td>{item.price * item.quantity}</td>
+                  </Item>
+                ))}
+              </tbody>
+            </Table>
+          ) : (
+            <p>You haven't added any item to the cart.</p>
+          )}
+        </CartListContainer>
+      </div>
+      {!!items.length && (
+        <Summary>
+          <h3>Summary</h3>
+          <hr />
+          <p>Items: {items.length}</p>
+          <p>Total cost: ${total}</p>
```

Comment:
I think you did an excellent job, just help me to correct that the maximum number of decimals is 2 digits

![Screen Shot 2022-07-18 at 13 17 14](https://user-images.githubusercontent.com/39224352/179576734-cca4b64a-a5b5-4875-9de0-f29f0d6f03f6.png)

If you see, when it is more than 2 digits the CSS is modified

![Screen Shot 2022-07-18 at 13 20 03](https://user-images.githubusercontent.com/39224352/179576934-67926f80-664b-4e28-af29-696f9f12ebd4.png)